### PR TITLE
Smalls addons for nginx config

### PR DESCRIPTION
**Pull request by dcaillibaud**
Adding some fastcgi parameter to nginx config (and a link to wwsympa.service sample) for spawn-fcgi

### CODE REVIEW

**Comment by ikedas** [comment]
As of 6.2.56, wwsympa became splitting URI by itself: See sympa-community/sympa#879 for details. That's why description about `split_path_info` was removed from example: It was moved to the note below.

As you wrote, CSS path may be required.  I'll check it.

Thanks for inputs!

**Comment by dcaillibaud** [comment]
Sorry, I didn't see theses notes (I'm still in 6.2.40 with debian buster and these lines are required, as note says).
I don't know if you can exclude the second commit from this PR, to keep the link added in the first commit.

If not I let you close the PR without merge, sorry for the noise.

**Comment by racke** [comment]
The CSS part is only needed for Debian installations.

**Comment by ikedas** [comment]
Hi @dcaillibaud ,

> As you wrote, CSS path may be required. I'll check it.

[`$CSSDIR`](https://sympa-community.github.io/manual/layout.html#cssdir) and [`$PICTURESDIR`](https://sympa-community.github.io/manual/layout.html#picturesdir) were subdirectories of [`$STATICDIR`](https://sympa-community.github.io/manual/layout.html#staticdir) by default, on version prior to 6.2.26. However recently they are vary by environments (check the links of three).  For purpose of example, I think all of these three would be better to appear in description.

----
BTW, to include your changes, could you please include addition of your name to [/AUTHORS.md](https://github.com/sympa-community/sympa-community.github.io/blob/master/AUTHORS.md) file into this PR?  It is required by [the license](https://sympa-community.github.io/LICENSE.html) applied to this documentation (CC-BY-SA 4.0).

File: manual/install/configure-http-server-spawnfcgi.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Note:
   1. Register WWSympa FastCGI service.
 
      Put ``wwsympa.service`` file into Systemd system directory
-     (such as ``/usr/lib/systemd/system``).
+     (such as ``/usr/lib/systemd/system``, see [theses examples](configure-http-server-systemdsocket.md#install-wwsympa-fastcgi-service)).
```
Comment (ikedas) [inline manual/install/configure-http-server-spawnfcgi.md:76]: This link points the file not using spawn-fcgi.

File: manual/install/configure-http-server-spawnfcgi.md [hunk 2]
```diff
@@ -207,13 +207,19 @@ instruction below.
...
          }
 
          location /static-sympa {
              alias $STATICDIR;
          }
+     
+         # needed for debian buster
+         location /css-sympa {
+             alias /var/lib/sympa/css;
+         }
```
Comment (ikedas) [inline manual/install/configure-http-server-spawnfcgi.md:222]: Detailed description will be added by PR #89.

**Comment by ikedas** [comment]
Hi @dcaillibaud , thanks for proposal of improvements.  I included most of them into my PR and merged it.  If you noticed anything, please submit changes again.

Thank you for input!

**Pull request closed without merging**